src/ext_slice.rs

```diff
@@ -3063,11 +3063,8 @@ pub trait ByteSlice: Sealed {
         // Finally, we are only dealing with u8 data, which is Copy, which
         // means we can copy without worrying about ownership/destructors.
         unsafe {
-            ptr::copy(
```

Comment:
Does miri complain here? I can't spot any problem with the existing code.

(Note that I'm aware of stacked borrows and understand the issues here. But that understanding came long after bstr was originally written.)

Reply:
Yes, Miri complains here: https://miri.saethlin.dev/ub?crate=bstr&version=1.0.0-pre.2